.github/workflows/_dist.yml

```diff
@@ -24,11 +27,12 @@ jobs:
           path: dist
 
       - name: Check for packaging errors
-        run: pipx run twine check --strict dist/*
+        run: uvx twine check --strict dist/*
 
       - name: Install produced wheel
         uses: ./.github/actions/install_requirements
         with:
+          # TODO: figure out what's going on
```

Comment:
I haven't checked dists yet, what problems do you see?

Reply:
None but I had no idea what to look for. The comment was more that I had no idea